client/layout/logged-out-design.jsx

```diff
@@ -9,9 +9,11 @@ import React from 'react';
  * Internal dependencies
  */
 import MasterbarLoggedOut from 'layout/masterbar/logged-out';
+import ThemesHead from 'my-sites/themes/head';
 
-const LayoutLoggedOutDesign = () => (
+const LayoutLoggedOutDesign = ( { tier = 'all' } ) => (
 	<div className="wp is-section-design has-no-sidebar">
+		<ThemesHead tier={ tier } />
```

Comment:
Yes. I wanted to have _something_ to render on the server, even if just temporarily, as PoC. Once all of `/design` becomes SSR-ready, we can obviously stick to just one Head component, likely close to or in the controller.